web/ui/react-app/src/pages/Status.tsx

```diff
@@ -1,6 +1,82 @@
-import React, { FC } from 'react';
+import React, { FC, Fragment } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Table, Alert } from 'reactstrap';
+import useFetch from './useFetch';
 
-const Status: FC<RouteComponentProps> = () => <div>Status page</div>;
+const ENDPOINTS = ['../api/v1/runtimeinfo', '../api/v1/buildinfo', '../api/v1/alertmanagers'];
+const sectionsTitles = ['Runtime Information', 'Build Information', 'Alertmanagers'];
 
-export default Status;
+interface StatusConfig {
+  [k: string]: { title: string; normalizeValue?: (v: any) => any };
+}
+
+type StatusPageState = Array<{ [k: string]: string }>;
+
+const normalizeAlertManagerValue = (alertMgrs: { url: string }[]) => {
```

Comment:
AlertManager -> Alertmanager (no logic behind it except being consistent with how we write Alertmanager :))